core/profiles/standard/standard.install

```diff
@@ -586,6 +586,7 @@ function standard_install() {
     'link_path' => 'user/login',
     'menu_name' => 'user-menu',
     'weight' => -1,
+    'customized' => 1,
```

Comment:
This flag may not be needed if `user_menu()` made the router item `user/login` into a `NORMAL_MENU_ITEM` instead of a `MENU_CALLBACK`